apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3329,18 +3329,18 @@
   "Message_BadWordsWhitelistDescription": "Add a comma-separated list of words to be removed from filter",
   "Message_Characther_Limit": "Message Character Limit",
   "Message_Code_highlight": "Code highlighting languages list",
-  "Message_Code_highlight_Description": "Comma separated list of languages (all supported languages at https://github.com/highlightjs/highlight.js/tree/9.18.5#supported-languages) that will be used to highlight code blocks",
+  "Message_Code_highlight_Description": "Comma separated list of languages (all supported languages at [highlight.js](https://github.com/highlightjs/highlight.js/tree/11.6.0#supported-languages)) that will be used to highlight code blocks",
   "Message_CustomDomain_AutoLink": "Custom Domain Whitelist for Auto Link",
   "Message_CustomDomain_AutoLink_Description": "If you want to auto link internal links like `https://internaltool.intranet` or `internaltool.intranet`, you need to add the `intranet` domain to the field, multiple domains need to be separated by comma.",
   "message_counter": "{{counter}} message",
   "message_counter_plural": "{{counter}} messages",
   "Message_DateFormat": "Date Format",
-  "Message_DateFormat_Description": "See also: <a href=\"http://momentjs.com/docs/#/displaying/format/\" target=\"momemt\">Moment.js</a>",
+  "Message_DateFormat_Description": "See also: [Moment.js](http://momentjs.com/docs/#/displaying/format/)",
   "Message_deleting_blocked": "This message cannot be deleted anymore",
   "Message_editing": "Message editing",
   "Message_ErasureType": "Message Erasure Type",
   "Message_ErasureType_Delete": "Delete All Messages",
-  "Message_ErasureType_Description": "Determine what to do with messages of users who remove their account.\n\n**Keep Messages and User Name:** The message and files history of the user will be deleted from Direct Messages and will be kept in other rooms.\n\n**Delete All Messages:**  All messages and files from the user will be deleted from the database and it will not be possible to locate the user anymore.\n\n**Remove link between user and messages:** This option will assign all messages and fles of the user to Rocket.Cat bot and Direct Messages are going to be deleted.",
+  "Message_ErasureType_Description": "Determine what to do with messages of users who remove their account.  \n -  **Keep Messages and User Name:** The message and files history of the user will be deleted from Direct Messages and will be kept in other rooms.  \n - **Delete All Messages:**  All messages and files from the user will be deleted from the database and it will not be possible to locate the user anymore.  \n - **Remove link between user and messages:** This option will assign all messages and fles of the user to Rocket.Cat bot and Direct Messages are going to be deleted.",
```

Comment:
```suggestion
  "Message_ErasureType_Description": "Determine what to do with messages of users who remove their account.  \n -  **Keep Messages and User Name:** The message and files history of the user will be deleted from Direct Messages but will be kept in other rooms.  \n - **Delete All Messages:**  All messages and files from the user will be deleted from the database and it will not be possible to locate the user anymore.  \n - **Remove link between user and messages:** This option will assign all messages and files of the user to Rocket.Cat bot and Direct Messages are going to be deleted.",
```